File: Ryhmätyö_kokeiluja_JAVALLA/Darts_peli/src/dartsPeli/peliMode.java
package dartsPeli;

public class peliMode {

    private int pelinPituus;
    private int pitkäPeli = 501;
    private int lyhytPeli = 301;



    public int annaPelinPituus(String pelinPituus){
     if (pelinPituus.equals("l")){
         this.pelinPituus = lyhytPeli;
     } else if (pelinPituus.equals("p")) {
         this.pelinPituus = pitkäPeli;
     } else {
         System.out.println("Annettua muoto ei ole olemassa");
     }
    return this.pelinPituus;
    }

    public String toString() {
        return "Valittu peli pelataan " + this.pelinPituus + " pisteestä.";
    }
}

File: IWD_RTYÖ/Pelimode.js
class PeliMode {
  constructor() {
    this.pelinPituus = 0;
    this.lyhytPeli = 301;
    this.pitkaPeli = 501;
  }

  annaPelinPituus(pelinPituus) {
    if (pelinPituus === "l") {
      this.pelinPituus = this.lyhytPeli;
    } else if (pelinPituus === "p") {
      this.pelinPituus = this.pitkaPeli;
    } else {
      console.log("Annettua muoto ei ole olemassa");
    }
    return this.pelinPituus;
  }

  toString() {
    return `Valittu peli pelataan ${this.pelinPituus} pisteestä.`;
  }
}

module.exports = PeliMode;